Source/CarthageKitTests/CartfileSpec.swift

```diff
@@ -74,23 +74,26 @@ class CartfileSpec: QuickSpec {
 		}
 
 		it("should detect duplicate dependencies in a single Cartfile") {
-			let testCartfileURL = Bundle(for: type(of: self)).url(forResource: "DuplicateDependencies/Cartfile", withExtension: "")!
+			let testCartfileURL = Bundle(for: type(of: self)).url(forResource: "DuplicateDependenciesCartfile", withExtension: "")!
 			let testCartfile = try! String(contentsOf: testCartfileURL, encoding: .utf8)
 
 			let result = Cartfile.from(string: testCartfile)
-			expect(result.error).to(beNil())
-
-			let cartfile = result.value!
-			expect(cartfile.dependencies.count) == 11
-
-			let dupes = cartfile.duplicateProjects().sort { $0.description < $1.description }
-			expect(dupes.count) == 2
-
-			let self2Dupe = dupes[0]
-			expect(self2Dupe) == ProjectIdentifier.gitHub(Repository(owner: "self2", name: "self2"))
-
-			let self3Dupe = dupes[1]
-			expect(self3Dupe) == ProjectIdentifier.gitHub(Repository(owner: "self3", name: "self3"))
+			expect(result.error).notTo(beNil())
+
+			if case let .duplicateDependencies(dupes)? = result.error {
+				let dupes = dupes
+					.map { $0.project }
+					.sort { $0.description < $1.description }
+				expect(dupes.count) == 2
+				
+				let self2Dupe = dupes[0]
+				expect(self2Dupe) == ProjectIdentifier.gitHub(Repository(owner: "self2", name: "self2"))
+				
+				let self3Dupe = dupes[1]
+				expect(self3Dupe) == ProjectIdentifier.gitHub(Repository(owner: "self3", name: "self3"))
+			} else {
+				fatalError("should error")
```

Comment:
I wasn't sure if there was a better way to do this with Quick/Nimble. If we were using XCTest directly, I'd use `XCFail`. But this certainly works. 😕